src/lib/domutil.js

```diff
@@ -0,0 +1,82 @@
+import helper from '@/lib/helper';
+
+const domutil = (function () {
+  let addEvent = null;
+  let removeEvent = null;
+  let element = null;
+  let listeners = [];
+
+  (function () {
+    // on
+    if (document.addEventListener) {
+      addEvent = function (type, handler) {
+        listeners.push(handler);
+        element.addEventListener(type, handler);
+      };
+    } else if (document.attachEvent) {
+      addEvent = function (type, handler) {
+        handler = handler.bind(element);
+
+        let reverseListeners = listeners.slice().reverse();
```

Comment:
const로 선언되어도 될 것 같습니다!

```suggestion
        const reverseListeners = listeners.slice().reverse();
```

Reply:
그러네요! 디테일한 부분 잡아주셔서 감사합니다!

Reply:
그러네요. 디테일한 부분 잡아주셔서 감사합니다!